lblod/frontend-gelinkt-notuleren
# Feat/manual voting

### Overview
Added a way to add a custom voting to the behandeling that will be just a document

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4991
https://github.com/lblod/app-gelinkt-notuleren/pull/198



### Setup
You should use the https://github.com/lblod/app-gelinkt-notuleren/pull/198 as backend

### How to test/reproduce
Go to the meeting form, and notice that you should be able to add custom votings that just let you write some text in the a document

### Challenges/uncertainties
We should maybe add some extra rdfa in the document, I will have a look when I finish the prepublisher and see how the final document looks like
As I said the prepublisher is not finished, so that's something I have to finish before merging this PR



### Checks PR readiness
